components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -975,14 +975,27 @@ private void sendResponse(HttpServletRequest req, HttpServletResponse resp, Stri
                               String tenantDomain)
             throws ServletException, IOException, IdentityException {
 
+        String spName = "";
         acUrl = getACSUrlWithTenantPartitioning(acUrl, tenantDomain);
+        String issuer = SAMLSSOUtil.getIssuerWithQualifierInThreadLocal();
 
         if (acUrl == null || acUrl.trim().length() == 0) {
             // if ACS is null. Send to error page
             log.error("ACS Url is Null");
             throw IdentityException.error("Unexpected error in sending message out");
         }
 
+        if (issuer != null && tenantDomain != null && !issuer.isEmpty() && !tenantDomain.isEmpty()){
```

Comment:
lets use StringUtils.isNotBlank
```suggestion
        if (StringUtils.isNotBlank(issuer) && StringUtils.isNotBlank(tenantDomain)){
```